package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```

Comment:
please revert this :P

Reply:
not even sure how this happened. Removed